<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">   background-color:#373d5b;</string>
  </property>
  <widget class="QTabWidget" name="tabview">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget#tabview {
    background-color: #075da8;
    
}

QTabWidget::pane {
    border: none;
}

QTabBar::tab {
    background-color: #075da8;
    color: white;
    padding: 8px;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
}

QTabBar::tab:selected {
    background-color: white;
    color: #075da8;
}

QTabBar::tab:hover {
    background-color: #1478c9;
    color: white;
}
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Mailer</string>
    </attribute>
    <widget class="QLineEdit" name="sender_email">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="placeholderText">
      <string>Sender Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="delay_time">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="placeholderText">
      <string>Delay time (seconds)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="start_after">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="placeholderText">
      <string>Start after (e.g: 1h 20m)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="subject">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Subject</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="body">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>431</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="placeholderText">
      <string>Body</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>370</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-bottom:3px solid #dc8c93;
border-radius:6px;

color:white;
background:rgb(42, 47, 70);</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pause">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>370</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom:3px solid #69b8ec;
border-radius:6px;

color:white;
background:rgb(42, 47, 70);</string>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pause_here">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="placeholderText">
      <string>Pause Here</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resume">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>370</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom:3px solid #69b8ec;
border-radius:6px;

color:white;
background:rgb(42, 47, 70);</string>
     </property>
     <property name="text">
      <string>resume</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="recipient_name">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="placeholderText">
      <string>Recipeint Name</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Add SMTP SERVER</string>
    </attribute>
    <widget class="QLineEdit" name="server">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="placeholderText">
      <string>Server</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_name">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="placeholderText">
      <string>User name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="placeholderText">
      <string>password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="port">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="placeholderText">
      <string>port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addServer">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>220</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom:3px solid #69b8ec;
border-radius:6px;

color:white;
background:rgb(42, 47, 70);</string>
     </property>
     <property name="text">
      <string>Add Server</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
